miden-tx/tests/integration/scripts/p2id.rs

```diff
@@ -105,16 +106,17 @@ fn prove_p2id_script() {
     let malicious_account =
         get_account_with_default_account_code(malicious_account_id, malicious_pub_key, None);
 
-    let data_store_malicious_account =
-        MockDataStore::with_existing(Some(malicious_account), Some(vec![note]));
+    let data_store_malicious_account = TransactionContextBuilder::new(malicious_account)
```

Comment:
nit: this should probably be named `tx_context_malicious_account`.

Reply:
done